docs/user-guide/federation/federated-services.md

```diff
@@ -0,0 +1,234 @@
+```
+eu.nginx.acme.com        CNAME nginx.mynamespace.myfederation.svc.europe-west1.example.com.
+us.nginx.acme.com        CNAME nginx.mynamespace.myfederation.svc.us-central1.example.com.
+nginx.acme.com             CNAME nginx.mynamespace.myfederation.svc.example.com.
+```
+That way your clients can always use the short form on the left, and always be automatcally routed to the closest healthy shard on their home continent.  All of the required failover is handled for you automatically by Kubernetes Cluster Federation.  Future releases will improve upon this even further.
+
+## Handling Failures of Backend Pods and Whole Clusters
+
+Standard Kubernetes service cluster-IP's already ensure that non-responsive individual Pod endpoints are automatically taken out of service with low latency (a few seconds). In addition, as aluded above, the Kubernetes Cluster Federation system automatically monitors the health of clusters and the endpoints behind all of the shards of your Federated Service, taking shards in and out of service as required (e.g. when all of the endpoints behind a service, or perhaps the entire cluster or availability zone go down, or conversely recover from an outage).  Due to the latency inherent in DNS caching (the cache timeout, or TTL for Federated Service DNS records is configured to 3 minutes, by default, but can be adjusted), it may take up to that long for all clients to completely fail over to an alternative cluster in in the case of catastrophic failure. However, given the number of discrete IP addresses which can be returned for each regional service endpoint (see e.g. us-central1 above, which has three alternatives) many clients will fail over automatically to one of the alternative IP's in less time than that given appropriate configuration.
+
+## Troubleshooting
+
+#### I cannot connect to my Cluster Federation API
+Check that your
+
+1. Client (typically kubectl) is correctly configured (including API endpoints and login credentials), and
+2. Cluster Federation API server is running and network-reachable.
+
+See the [federation admin guide](/docs/admin/federation/) to learn
+how to bring up a cluster federation correctly (or have your cluster administrator do this for you), and how to correctly configure your client.
+
+#### I can create a Federated Service successfully against the Cluster Federation API, but no matching services are created in my underlying clusters
+Check that:
+
+1. Your clusters are correctly registered in the Cluster Federation API (`kubectl describe clusters`)
+2. Your clusters are all 'Active'.  This means that the cluster Federation system was able to connect and authenticate against the clusters' endpoints.  If not, consult the logs of the federation-controller-manager pod to ascertain what the failure might be. (`kubectl logs federation-controller-manager --namespace federation`)
```

Comment:
The pod isn't actually named `federation-controller-manager`.

`kubectl --namespace=federation logs $(kubectl get pods --namespace=federation -l module=federation-controller-manager -oname)`